app/assets/stylesheets/events.scss

```diff
@@ -26,4 +26,9 @@
 .event-image {
   max-width: 500px;
 }
-
+#spinner {
```

Comment:
for css use classes instead of ids :) id selector has higher priority which means they are harder to override